features/bevragen/persoon-beperkt/overlijden/overzicht.feature

```diff
@@ -0,0 +1,129 @@
+# language: nl
+
+Functionaliteit: Persoon beperkt: overlijden
+
+  Wanneer een consumer van de BRP API personen zoekt inclusief overleden personen
+  Dan wil de consumer worden geattendeerd welke van de gevonden personen zijn overleden zonder expliciet om overlijden gegevens te vragen met de fields parameter
+
+  Rule: indicatieOverleden heeft de waarde true als een gevonden persoon is overleden. Dit veld wordt altijd meegeleverd, ook als het niet met fields is gevraagd
+
+    Abstract Scenario: <sub titel> van een niet-overleden persoon
+      Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens
+      | geslachtsnaam (02.40) | geboortedatum (03.10) |
+      | Jansen                | 19630405              |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                       | waarde                              |
+      | type                       | ZoekMetGeslachtsnaamEnGeboortedatum |
+      | geslachtsnaam              | Jansen                              |
+      | geboortedatum              | 1963-04-05                          |
+      | inclusiefOverledenPersonen | true                                |
+      | fields                     | <fields>                            |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000152 |
+
+      Voorbeelden:
+      | sub titel                             | fields                         |
+      | overlijden velden wordt gevraagd      | burgerservicenummer,overlijden |
+      | overlijden velden wordt niet gevraagd | burgerservicenummer            |
+
+    Abstract Scenario: overlijden gegevens wordt niet gevraagd van een overleden persoon
+      Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens
+      | geslachtsnaam (02.40) | geboortedatum (03.10) |
+      | Jansen                | 19630405              |
+      En de persoon heeft de volgende 'overlijden' gegevens
+      | datum overlijden (08.10) | plaats overlijden (08.20) | land overlijden (08.30) |
+      | <datum overlijden>       | 0518                      | 6030                    |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                       | waarde                              |
+      | type                       | ZoekMetGeslachtsnaamEnGeboortedatum |
+      | geslachtsnaam              | Jansen                              |
+      | geboortedatum              | 1963-04-05                          |
+      | inclusiefOverledenPersonen | true                                |
+      | fields                     | burgerservicenummer                 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                          | waarde    |
+      | burgerservicenummer           | 000000152 |
+      | overlijden.indicatieOverleden | true      |
```

Comment:
@CathyDingemanse hadden we niet besloten indicatieOverleden toch niet ongevraagd mee te leveren, omdat er ook opschorting bijhouding met waarde overleden wordt meegeleverd?